src/main/java/com/meetup/server/place/implement/PlaceSorter.java

```diff
@@ -26,4 +28,16 @@ public List<PlaceResponse> sortByRating(List<PlaceResponse> placeResponses) {
         mergedPlaceResponses.addAll(placeResponsesWithoutReview);
         return mergedPlaceResponses;
     }
+
+    public List<Review> sortSpotReviewsByDesc(List<Review> reviews) {
```

Comment:
```suggestion
    public List<Review> sortSpotReviewsByNewest(List<Review> reviews) {
```
어떤 값을 기준으로 내림차순 정렬인지 알기 모호해서 위와 같은 식으로 명확히 알 수 있게 바꾸면 좋을 거 같아요!